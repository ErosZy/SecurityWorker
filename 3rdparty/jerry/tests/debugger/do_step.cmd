step
step
backtrace
next
bt
next
s
n
bt
c
