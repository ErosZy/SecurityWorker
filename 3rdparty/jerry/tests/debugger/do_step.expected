Connecting to: localhost:5001
Stopped at tests/debugger/do_step.js:32
(jerry-debugger) step
Stopped at tests/debugger/do_step.js:22 (in f1() at line:15, col:1)
(jerry-debugger) step
Stopped at tests/debugger/do_step.js:19 (in g() at line:17, col:3)
(jerry-debugger) backtrace
Frame 0: tests/debugger/do_step.js:19 (in g() at line:17, col:3)
Frame 1: tests/debugger/do_step.js:22 (in f1() at line:15, col:1)
Frame 2: tests/debugger/do_step.js:32
(jerry-debugger) next
Stopped at tests/debugger/do_step.js:23 (in f1() at line:15, col:1)
(jerry-debugger) bt
Frame 0: tests/debugger/do_step.js:23 (in f1() at line:15, col:1)
Frame 1: tests/debugger/do_step.js:32
(jerry-debugger) next
Stopped at tests/debugger/do_step.js:33
(jerry-debugger) s
Stopped at tests/debugger/do_step.js:28 (in f2() at line:26, col:1)
(jerry-debugger) n
Stopped at tests/debugger/do_step.js:29 (in f2() at line:26, col:1)
(jerry-debugger) bt
Frame 0: tests/debugger/do_step.js:29 (in f2() at line:26, col:1)
Frame 1: tests/debugger/do_step.js:33
(jerry-debugger) c
